README.md

```diff
@@ -0,0 +1,35 @@
+# Open Pension Scraper
+
+[![Build Status][travis-image]][travis-url]
+
+This repo contains tools and [meta]data for the purpose of extracting publicly-available
+online information to be used by Open Pension.
+
+Open Pension is a "Hasadna" project, aiming to reveal the secrets behind the Israeli pension market.
+
+#### Note to contributors
```

Comment:
- Add a space `enter` between the title and the text.
- The title should be `h2` (`##`), because it's right after `h1`.